lib/ansible/modules/identity/ipa/ipa_sudorule.py

```diff
@@ -308,6 +333,8 @@ def main():
                          host=dict(type='list', required=False),
                          hostcategory=dict(type='str', required=False, choices=['all']),
                          hostgroup=dict(type='list', required=False),
+                         runasusercategory=dict(type='str', required=False, choices=['all']),
```

Comment:
Required is by-default False, so need to mention explicitly. Can you please remove the rest of them as well ?

Reply:
Done!